static/coffee/directives.coffee

```diff
@@ -30,3 +28,27 @@ directives.directive('cpContact', () ->
     ]      
   }
 )
+
+#----------------------------------------------------------------------------
+# A contact field value
+#----------------------------------------------------------------------------
+directives.directive('cpFieldvalue', () ->
```

Comment:
@justinnvdm have added this to do formatting of date and numerical values, as we're currently just dumping ISO8601 timestamps on unfortunate users. Logic below should probably be shared in some way with pod values.